content/md/what-is/what-is-an-asset.md

```diff
@@ -0,0 +1,53 @@
+---
+id: what-is-an-asset
+themes: first-steps, akeneo-concepts, manage-your-assets
+title: What is an **asset**?
+popular: false
+ee-only: true
+related: assets-transformation, upload-assets, work-with-assets
+---
+
+# Definition of an asset
+
+An asset can be an HD image, a video, a PDF installation guide... Any kind of file that can be used to enrich a product in Akeneo.
+
+::: ee
+Assets are only available in the Enterprise Edition.
+:::
+
+An asset is always `scopable` because you made need asset reference file variations for each channel - for example, a full size image for print and a small image for ecommerce.  
+It can also be `localisable`, if you want to have a specific version for the locales enabled in the PIM. For instance, you can have a PDF guide in English, another one in French, another one in German...
+Thus, for each asset created, you may have several files attached to it.
+
+![Asset](../img/Settings_What-is-an-asset.svg)
+
+An asset has a unique code which is mandatory, and optional properties:
+- One or more reference files,
+- One variation (or none) per channel,
+- A description,
+- An end date of use,
+- Some tags (to qualify the asset and find it quickly).
+
+To unleash the power of our Product Asset Manager (PAM), you will need to upload asset transformations for your channels (see our dedicated : [About the asset transformations](assets-transformation.html) so the PIM generates variations based on the reference file.  
+Let's say you upload an HD image as a reference file, you can decide to reduce its resolution and/or size for your e-commerce channel, and do the same for the mobile channel. This is what asset transformation is used for.
```

Comment:
last sentence: maybe say (use plural instead of singular): "This is what asset transformations are used for." ?

Reply:
```suggestion
Let's say you upload an HD image as a reference file, you can decide to reduce its resolution and/or size for your e-commerce channel, and do the same for the mobile channel. This is what asset transformations are used for.
```